### PR TITLE
Japanese translation part2

### PR DESCRIPTION
**Pull request by Yamagami-Takuma**

### CODE REVIEW

**Comment by Yamagami-Takuma** [comment]
@f-higashi  
Can you review?

File: i18n/messages-ja.xtb [hunk 1]
```diff
@@ -1298,39 +1298,39 @@
...
+  <translation id="6128908853633901897" key="MSG_REPLICATIONCONTROLLERDETAIL_REPLICATIONCONTROLLERINFO_9" desc="The message says how many pods are pending (replication controller details page).">{{::$ctrl.replicationController.podInfo.pending}} 待機中</translation>
   <translation id="492720185165061432" key="MSG_REPLICATIONCONTROLLERDETAIL_UPDATEREPLICAS_0" desc="Title for the pod count update dialog (for a replication controller).">ポッド要求数の設定</translation>
-  <translation id="3157265907543577974" key="MSG_REPLICATIONCONTROLLERDETAIL_UPDATEREPLICAS_1" desc="User help for the pod count update dialog (on the replication controllers detail page).">Replication controller {{::$ctrl.replicationController}} will be updated to reflect the desired count.</translation>
+  <translation id="3157265907543577974" key="MSG_REPLICATIONCONTROLLERDETAIL_UPDATEREPLICAS_1" desc="User help for the pod count update dialog (on the replication controllers detail page).">レプリケーションコントローラ {{::$ctrl.replicationController}} は要望数を反映して更新されます。</translation>
```
Comment (f-higashi) [inline i18n/messages-ja.xtb]: You  already translated `desired`  to `要求`.

File: i18n/messages-ja.xtb [hunk 2]
```diff
@@ -1409,24 +1409,24 @@
...
+  <translation id="3101823090504084917" key="MSG_RESOURCELIMIT_RESOURCELIMITS_3" desc="]Resource Limits min entry.">最小</translation>
+  <translation id="3535089658933789537" key="MSG_RESOURCELIMIT_RESOURCELIMITS_4" desc="Resource Limits max entry.">最大</translation>
+  <translation id="5393078879716677215" key="MSG_RESOURCELIMIT_RESOURCELIMITS_4" desc="Resource Limits default request entry.">デフォルトリクエスト</translation>
+  <translation id="6070144888655765726" key="MSG_RESOURCELIMIT_RESOURCELIMITS_5" desc="Resource Limits default limit entry.">デフォルト上限</translation>
+  <translation id="6947338869559428640" key="MSG_RESOURCELIMIT_RESOURCELIMITS_5" desc="Resource Limits default request entry.">デフォルトリクエスト</translation>
+  <translation id="836896690388641861" key="MSG_RESOURCELIMIT_RESOURCELIMITS_6" desc="Resource Limits default limit entry.">デフォルト上限</translation>
+  <translation id="5650624948356810172" key="MSG_RESOURCELIMIT_RESOURCELIMITS_6" desc="Resource Limits max limit/request ratio entry.">上限/リクエスト率</translation>
+  <translation id="323516129286855678" key="MSG_RESOURCELIMIT_RESOURCELIMITS_7" desc="Resource Limits max limit/request ratio entry.">上限/リクエスト率</translation>
```
Comment (f-higashi) [inline i18n/messages-ja.xtb]: `率` is wrong.

In this case, I think only `最大上限/リクエスト` .

**Comment by Yamagami-Takuma** [comment]
@f-higashi  
I fixed them!

**Comment by Yamagami-Takuma** [comment]
And I added 2 lines, that appeared in xtb file when `gulp serve:prod` executed.
* https://github.com/kubernetes/dashboard/pull/1494/commits/32d133fa117f874c7e39d05ae61a6cd3bd5b6c54#diff-eca4a5d4b29e9961550ce6b3bc583a82R714
* https://github.com/kubernetes/dashboard/pull/1494/commits/32d133fa117f874c7e39d05ae61a6cd3bd5b6c54#diff-29c15e3cf792eb8d814cfc58b5ae3d14R1445

**Comment by Yamagami-Takuma** [comment]
Fix-up is done.

**Pull request merged**